### PR TITLE
Update docstring of load_all_gen_kw_data

### PR DESCRIPTION
**Pull request by dafeda**
Don't calculate ens_mask if realization_index is not None.

- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: src/ert/storage/local_ensemble.py [hunk 1]
```diff
@@ -440,32 +441,34 @@ def load_all_gen_kw_data(
...
-        Returns:
-            DataFrame: A pandas DataFrame containing the GEN_KW data.
-
-        Raises:
-            IndexError: If a non-existent realization index is provided.
-
-        Note:
+        """Loads scalar parameters (GEN_KWs) into a pandas DataFrame
+        with columns <PARAMETER_GROUP>:<PARAMETER_NAME> and
+        and "Realization" as index.
```
Comment (Blunde1) [inline src/ert/storage/local_ensemble.py:446]: Is there a double "and" here?

Comment (dafeda) [inline src/ert/storage/local_ensemble.py:446]:
```suggestion
        "Realization" as index.
```

Comment (dafeda) [reply src/ert/storage/local_ensemble.py]: Yes, fixed.

**Review by Blunde1 — APPROVED** [review]
Thanks! LGTM

**Review by frode-aarstad — APPROVED** [review]

**Pull request merged**